.github/workflows/build.yml

```diff
@@ -12,3 +12,28 @@ jobs:
         run: npm run lint
       - name: Run tests in all packages
         run: npm run test
+  release-alpha:
+    needs: build
+    if: github.ref_name == 'main'
+    runs-on: ubuntu-latest
+    container: cypress/browsers:node14.17.0-chrome91-ff89
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.SUPPORT_TOKEN }}
+      - name: Install lerna and all packages
+        run: npm ci
```

Comment:
I know Ivan said not to, but I think we should avoid running the build twice. To do so, we can create a custom action (instead of a reusable workflow) that does all the setup and release.

Then this custom action can be invoked from the release-alpha (with no conditions) and build (only if it is on the `main` branch) workflows, passing to it the needed NPM and git tokens.

This way we don't need to upload artifacts, neither to do 2 builds.